data_diff/__main__.py

```diff
@@ -44,7 +44,12 @@
 @click.option("-v", "--verbose", is_flag=True, help="Print extra info")
 @click.option("-i", "--interactive", is_flag=True, help="Confirm queries, implies --debug")
 @click.option(
-    "-j", "--threads", default=None, help="Number of threads to use. 1 means no threading. Auto if not specified."
+    "-j",
+    "--threads",
+    default="1",
+    help="Number of worker threads to use per database. Default=1. "
```

Comment:
Would it be worth adding some more information here about tuning of this?

```
"Number of worker threads to use per database to execute queries simultaneously. Default = 1"
"A higher number will increase performance, but take more capacity from your database."
"Tune this while monitoring your database."
```

Reply:
It's definitely good information to provide. The question is should this be part of the help text, or somewhere in the docs?

Reply:
I think at least the first line in the help text, if you think it's all too long.